### ISSUE TITLE
Windows 10 ANSI support

### ISSUE THREAD

**Issue by TheOneRing** [xolox/python-coloredlogs#71]
Windows 10 supports ANSI by default.
But right now the use of Colorama on Windows is hardcoded.
 `terminal.terminal_supports_colors` looks for the env var ANSICON maybe the same can be used for coloredlogs?

**Comment by TheOneRing** [comment]
This would allow logging binary streams (including other ansi codes)

**Comment by TheOneRing** [comment]
ANSI support gets enabled by calling
```
        ctypes.windll.kernel32.SetConsoleMode(ctypes.windll.kernel32.GetStdHandle(-11), 7)
        ctypes.windll.kernel32.SetConsoleMode(ctypes.windll.kernel32.GetStdHandle(-12), 7)
```

**Comment by xolox** [comment]
Hi Hannah, thanks for the suggestion and sorry for the late response. I've just released [coloredlogs 14.0](https://coloredlogs.readthedocs.io/en/latest/changelog.html#release-14-0-2020-02-16) which knows how to enable native Windows ANSI support as suggested here. I don't use Windows so didn't have an environment available to test this in, but I was crazy enough to download a [Windows Edge test VM](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/) and it seems to be working well :grin:. Thanks for the suggestion!
